apps/emqx_connector_aggregator/src/emqx_connector_aggregator.app.src

```diff
@@ -0,0 +1,14 @@
+{application, emqx_connector_aggregator, [
+    {description, "EMQX Enterprise Connector Data Aggregator"},
+    {vsn, "0.1.0"},
+    {registered, []},
+    {applications, [
+        kernel,
+        stdlib,
+        emqx_resource
```

Comment:
Curious: why do we need to depend on `emqx_resource`?

Reply:
Good point, I think this app doesn't actually use it.  I'll remove it.